tests/apc_hugetlb.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Check apc.mmap_hugetlb_mode parameter
+--SKIPIF--
+<?php
+require_once(dirname(__FILE__) . '/skipif.inc'); 
+if (PHP_OS != "Linux") die("skip only on Linux");
```

Comment:
Is this test going to work on a system with huge pages disabled (without throwing any errors?)